cmd/tailscale/cli/status.go

```diff
@@ -69,7 +70,14 @@ var statusArgs struct {
 }
 
 func runStatus(ctx context.Context, args []string) error {
-	st, err := tailscale.Status(ctx)
+	if len(args) > 0 {
+		return errors.New("unexpected non-flag arguments to 'tailscale status'")
+	}
+	getStatus := tailscale.Status
+	if !statusArgs.peers {
```

Comment:
Where was this being used previously?

Reply:
there was some manual filtering later in this file.

but I have a change coming after this one that lets you see filter the status (including CLI+JSON) to specific peers and I'll unify the two then